tasks-schema.json

```diff
@@ -356,24 +466,175 @@
+                          ]
+                        }
+                      },
+                      "required": [
+                        "type_id",
+                        "value"
+                      ]
+                    },
+                    "categorical": {
+                      "description": "Object defining a categorical output type.",
+                      "type": "object",
+                      "properties": {
+                        "type_id": {
+                          "description": "Object containing required and optional arrays specifying valid category values.",
+                          "example": [
+                            ["Vaccinated", "Unvaccinated"]
+                          ],
+                          "type": "object",
+                          "properties": {
+                            "required": {
+                              "description": "Array of unique categories that must be present for submission to be valid. Can be null if no categories are required and all valid categories are specified in the optional property.",
+                              "type": ["array", "null"],
+                              "items": {
+                                "type": "string"
+                              }
+                            },
+                            "optional": {
+                              "description": "Array of valid but not required unique categories. Can be null if all categories are required and are specified in the required property.",
+                              "type": ["array", "null"],
+                              "items": {
+                                "type": "string"
+                              }
+                            }
+                          },
+                          "required": [
+                            "required",
+                            "optional"
+                          ]
+                        },
+                        "value": {
+                          "type": "object",
+                          "description": "Object defining valid values of the probability mass function of the predictive distribution for a given level of a categorical outcome variable.",
+                          "example": [0.4, 0.6],
+                          "properties": {
+                            "type": {
+                              "description": "Data type of the probability mass function values.",
+                              "example": "double",
+                              "enum": [
+                                "numeric", "integer", "double"
+                              ]
+                            },
+                            "minimum": {
+                              "description": "The minimum inclusive valid probability mass function value",
+                              "example": 0,
+                              "type": "number",
+                              "minimum": 0,
```

Comment:
Resolved in 0e44364ae4ba3295fc9b503681ad00bd7f1e5878

Note that I have removed the examples because the values are now a constant. Let me know if you think I need to reintroduce them